pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -217,7 +217,7 @@ in {
   zfsStable = common {
     # check the release notes for compatible kernels
     kernelCompatible = kernel.kernelOlder "5.20";
-    latestCompatibleLinuxPackages = linuxPackages_5_19;
+    latestCompatibleLinuxPackages = linuxPackages_5_15;
```

Comment:
:cry:

Reply:
@Ma27 What's the worst thing that can happen if we keep an EOL-ed kernel version in Nixpkgs, until a ZFS version is released that supports a newer, stable kernel version?

Reply:
How about requiring the use of `allowInsecurePredicate` to prevent users from unknowingly using an EOL-ed kernel?